tests/manage/z_cluster/test_must_gather.py

```diff
@@ -1,270 +1,44 @@
-        )
+    mustgather = MustGather()
+    mustgather.collect_must_gather()
 
-        # 2nd test: Verify logs file are not empty
-        logs_dir_list = self.search_log_files(directory)
-        assert self.check_file_size(logs_dir_list), (
-            "One or more log file are empty"
-        )
+    def teardown():
+        mustgather.cleanup()
+    request.addfinalizer(teardown)
+    return mustgather
 
-        # Find must_gather_commands directory for verification
-        for dir_root, dirs, files in os.walk(directory + "_ocs_logs"):
-            if os.path.basename(dir_root) == 'must_gather_commands':
-                logger.info(
-                    f"Found must_gather_commands directory - {dir_root}"
-                )
-                assert 'json_output' in dirs, (
-                    "json_output directory is not present in "
-                    "must_gather_commands directory."
-                )
-                assert files, (
-                    "No files present in must_gather_commands directory."
-                )
-                cmd_files_path = [
-                    os.path.join(dir_root, file_name) for file_name in files
-                ]
-                json_output_dir = os.path.join(dir_root, 'json_output')
-                break
 
-        # Verify that command output files are present as expected
-        assert set(constants.MUST_GATHER_COMMANDS).issubset(files), (
-            f"Actual and expected commands output files are not matching.\n"
-            f"Actual: {files}\nExpected: {constants.MUST_GATHER_COMMANDS}"
-        )
-        if sorted(constants.MUST_GATHER_COMMANDS_JSON) != sorted(files):
-            logger.warning(
-                "There are more actual must gather commands than expected"
-            )
-
-        # Verify that files for command output in json are present as expected
-        commands_json = os.listdir(json_output_dir)
-        assert set(constants.MUST_GATHER_COMMANDS_JSON).issubset(commands_json), (
-            f"Actual and expected json output commands files are not "
-            f"matching.\nActual: {commands_json}\n"
-            f"Expected: {constants.MUST_GATHER_COMMANDS_JSON}"
-        )
-        if sorted(constants.MUST_GATHER_COMMANDS_JSON) != sorted(commands_json):
-            logger.warning(
-                "There are more actual must gather commands than expected"
-            )
-
-        # Verify that command output files are not empty
-        empty_files = []
-        json_cmd_files_path = [
-            os.path.join(json_output_dir, file_name) for file_name in commands_json
+class TestMustGather(ManageTest):
+    @tier1
+    @pytest.mark.parametrize(
+        argnames=['log_type'],
+        argvalues=[
+            pytest.param(
+                *['CEPH'], marks=pytest.mark.polarion_id("OCS-1583")
+            ),
+            pytest.param(
+                *['JSON'], marks=pytest.mark.polarion_id("OCS-1583")
```

Comment:
Not sure how this will be reported to the polarion if in xunit file we will have 3 test cases with one ID. Maybe worth to check.

Reply:
@petr-balogh 
@AviadP asked me to split the must_gather test.
I think that it is good idea, because it will make the analysis process easier.
I dont think that I need to create new polarion id per type of file.. what do you think?

Reply:
I see your point, one thing I don't know if xunit file has 3 TC with the same polarion ID, if all will be reported to TC in polarion or only one of them and not sure which one, maybe only last?  You don't need to deal with it now, just wort figure it out.